docker-compose.yml

```
@@ -0,0 +1,17 @@
+name: ultrasinger
```

Should we add version?

```suggestion
---
version: "3.9"

name: ultrasinger
```

---

done